index.html

```diff
@@ -3542,14 +3543,14 @@ <h1 id="title" class="text-xl font-bold mx-auto truncate w-full">
       <!-- Start game -->
       <canvas
         id="canvas"
-        class="block fixed left-0"
+        class="block portrait:fixed left-0"
```

Comment:
this was it, the content was being hidden on the left behind the canvas on desktop because canvas is position fixed